src/test/java/com/zufar/onlinestore/security/api/CustomUserDetailsServiceTest.java

```diff
@@ -0,0 +1,57 @@
+package com.zufar.onlinestore.security.api;
+
+import com.zufar.onlinestore.user.entity.UserEntity;
+import com.zufar.onlinestore.user.repository.UserRepository;
+import org.instancio.Instancio;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class CustomUserDetailsServiceTest {
+    @InjectMocks
+    private CustomUserDetailsService customUserDetailsService;
+    @Mock
+    private UserRepository userRepository;
+    private String userEmail = Instancio.of(String.class)
+            .create();
```

Comment:
please leave method on the same line 
`private String userEmail = Instancio.of(String.class).create();`